build/tasks/CreateCommonManifestTask.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json.Linq;
+
+namespace RepoTasks
+{
+    public class CreateCommonManifestTask : Task
```

Comment:
Also, MSBuild convention is to leave off the `Task` name.  `public class CreateCommonManifest : Task`